grpc/cubic.proto

```diff
@@ -276,8 +273,25 @@ message RecognitionResult {
 // A recognition hypothesis
 message RecognitionAlternative {
   // Text representing the transcription of the words that the user spoke.
+  //
+  // The transcript will be formatted according to the servers formatting
+  // configuration. If you want the raw transcript, please see the field
+  // `raw_transcript`.
+  //
+  // As an example, if the spoken utterance was `here are four words`,
+  // this field would be set to "Here are 4 words".
```

Comment:
As an example, if the spoken utterance was `four people`, and the server was configured to format numbers, this field would be set to `4 people`.